eamena/eamena/models/forms.py

```diff
@@ -160,21 +160,34 @@ def get_info():
         return {
             'id': 'modification',
             'icon': 'fa-plus',
-            'name': _('Modifications'),
+            'name': _('Modification and Construction'),
             'class': ModificationForm
         }
 
     def update(self, data, files):
-        self.update_nodes('MODIFICATION.E11', data)
+        self.update_nodes('MODIFICATION_TYPE.E55', data)
+        self.update_nodes('MODIFICATION_DESCRIPTION.E62', data)
+        self.update_nodes('CONSTRUCTION_TECHNIQUE_TYPE.E55', data)
         return
 
     def load(self, lang):
         if self.resource:
-            self.data['MODIFICATION.E11'] = {
-                'branch_lists': self.get_nodes('MODIFICATION.E11'),
+            self.data['MODIFICATION_TYPE.E55'] = {
+                'branch_lists': datetime_nodes_to_dates(exclude_empty_branches(self.get_nodes('MODIFICATION.E11'),'MODIFICATION_TYPE.E55')),
                 'domains': {
-                    'CONSTRUCTION_TECHNIQUE_TYPE.E55' : Concept().get_e55_domain('CONSTRUCTION_TECHNIQUE_TYPE.E55'),
                     'MODIFICATION_TYPE.E55' : Concept().get_e55_domain('MODIFICATION_TYPE.E55'),
+                }
+            }
+            
+            self.data['MODIFICATION_DESCRIPTION.E62'] = {
+                'branch_lists': datetime_nodes_to_dates(exclude_empty_branches(self.get_nodes('MODIFICATION.E11'),'MODIFICATION_DESCRIPTION.E62')),
```

Comment:
@mradamcox I don't think `datetime_nodes_to_dates()` is needed here, as this branch does not contain any dates.